src/sentry/api/event_search.py

```diff
@@ -1922,6 +1922,38 @@ def result_type_fn(function_arguments, parameter_values):
             default_result_type="number",
         ),
         Function("failure_rate", transform="failure_rate()", default_result_type="percentage"),
+        Function(
+            "failure_count",
+            aggregate=[
+                "countIf",
+                [
+                    [
```

Comment:
could we use the same `has` function that we use for key_transactions here instead?

Reply:
Updated in 5dcb9e6